resources/images/model/images.go

```diff
@@ -341,6 +342,11 @@ func getMetaFromArchive(r *io.Reader) (*images.SoftwareImageMetaArtifactConstruc
 		return nil
 	}
 
+        aReader.ScriptsReadCallback = func(r io.Reader, fi os.FileInfo) error {
```

Comment:
Do we need this callback? - looks like piece of unused code that may stick around for a long time.